### PR TITLE
environment vars

### PR DESCRIPTION
**Pull request by DonDuala**
use cli env vars for sensitive constants

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,9 @@ Trading events per event:
 3. Buy or Sell delta to 0 via TWAP if necessary
 4. Place bid & ask at gamma levels
 5. Start Scalper Window timer
+
+Environment Variables
+DEV=true || DEV=false
+RPC=[Enter RPC url]
+SOL, BTC, ETH, MNGO = [OFF || ON,DELTA_OFFSET,THEO_VOL]
+Example: DEV=false RPC=https://api.mainnet-beta.solana.com SOL=ON,-117.7,0.84 BTC=OFF ETH=OFF MNGO=ON,0,1.6 yarn run main
```
Comment (brittcyr) [inline README.md:23]: now that vol is hidden, can we remove the randomness added in router when determining if a bid exceeds threshold?
does it make sense to have both uses of theo vol still be the same constant, or should they be different?

File: src/config.ts [hunk 2]
```diff
@@ -1,15 +1,39 @@
...
+
 export const productStatus = new Map<string, boolean> ([
-  ['BTC', false], ['ETH', false], ['SOL', true], ['MNGO', false]
+  ['BTC', btc_vars[0] == "ON" ? true : false], 
+  ['ETH', eth_vars[0] == "ON" ? true : false], 
+  ['SOL', sol_vars[0] == "ON" ? true : false], 
+  ['MNGO', mngo_vars[0] == "ON" ? true : false]
+]);
+// Adjust delta hedges for loans, negative values allow positive spot balances in mango
+// CAUTION! Turn off scalper, send funds to mango & update value before running!
```
Comment (brittcyr) [inline src/config.ts:18]: better than a warning here in the code would be to log the delta offset at the start of a scalper run, and if we think it is necessary, add a roadblock of some kind.

File: README.md [hunk 3]
```diff
@@ -15,3 +15,9 @@ Trading events per event:
 3. Buy or Sell delta to 0 via TWAP if necessary
 4. Place bid & ask at gamma levels
 5. Start Scalper Window timer
+
+Environment Variables
+DEV=true || DEV=false
```
Comment (brittcyr) [inline README.md:20]: mention that this defaults to DEV=true

**Review by brittcyr — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (DonDuala) [reply README.md:23]: We still want it or at least I don't see an issue keeping it. The implied vol can still be backed out. 

I am thinking that this threshold price we may want to use the Backstop IV not Theo IV however. As it's unlikely that an MM will bid above what the theo IV in the market. Probably rename this parameter to HEDGE_VOL

File: src/config.ts (hunk 2 above)
Comment (DonDuala) [reply src/config.ts:18]: separated offset delta from spot delta and logging it now

Comment (DonDuala) [reply src/config.ts:18]: image

File: README.md (hunk 1 above)
Comment (DonDuala) [reply README.md:23]: https://github.com/Dual-Finance/risk-manager/issues/78

Comment (brittcyr) [reply README.md:23]: Not an issue, but if the randomness does become unnecessary, should be removed for the sake of code simplicity and maintainability. Good for now though.

**Pull request merged**